src/App.css

```diff
@@ -1,38 +1,47 @@
-.App {
-  text-align: center;
+/*Universal CSS*/
+
+header {
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #591902;
+}
+
+nav {
+  font: Arial, Helvetica, sans-serif;
+  color: #591902;
+}
+
+body {
+  background-color: #EBEDF2;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+h1 {
```

Comment:
Unless we plan on having the h1 be styled differently, you can combine header styles into one like so:
`h1, h2, h3, h4 {
  font-family: Garamond, sans-serif;
  color: #591902;
}`
Also, can you include down to h4? For now, the catering page is using up to h4. Might change later.